custom_components/powercalc/data/signify/LCB001/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Kasa, KP115(US)",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v0.24.8:docker"
+    },
+    "name": "\tHue White and Color Ambiance BR30 Bluetooth",
```

Comment:
Extra tab character? \t

Reply:
Very strange I don't see this when I edit the file in github browser.

Reply:
I had some c+p errors I cleaned up before submitting the PR. Odd that they still appear. They should have been resolved in https://github.com/bramstroker/homeassistant-powercalc/commit/e2e089a40978fb46db48f7a889a767825c02b0e7 and https://github.com/bramstroker/homeassistant-powercalc/commit/a1ea3944fe9b0f91d2284e9767d94acc7e00a895